docs/pages/desktop-access/reference/sessions.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Session Recording and Playback
+description: Recording and Playing Back Teleport Desktop Access Sessions.
+---
+
```

Comment:
Should this be a separate H1 section on disabling session recording?

If so, we'd need to move the first sentence of this next paragraph before the H1 header.